samples/sensor/accel_stream/README.rst

```diff
@@ -0,0 +1,10 @@
+.. _accel-stream-sample:
+
+Accel Streaming Sample
+######################
```

Comment:
please use `:zephyr:code-sample` directive 
see https://github.com/zephyrproject-rtos/zephyr/blob/ee451e5cf050285b850fdef8e6afaec666994928/doc/templates/sample.tmpl#L1-L6 and note the guidelines re: wording to use for the description

once you have the directive in place, you can directly reference the code sample using `` .... :zephyr:code-sample:`sample-id` ...  `` so the `.. _accel-stream-sample` anchor can be dropped.

recommended name for the sample: "Accelerometer sensor streaming"